main.js

```diff
@@ -0,0 +1,107 @@
+const gameBoard = (() => {
+  const initBoard = () => document.querySelectorAll('.gameboard').forEach(element => {
+      element.innerText = '';
+      element.classList.remove('clicked');
+  });
+  return {
+      initBoard
+  };
+})();
+
+const game = (() => {
+  const winCombo = [
+  ['1', '2', '3'], 
+  ['4', '5', '6'], 
+  ['7', '8', '9'], 
+  ['1', '4', '7'], 
+  ['2', '5', '8'], 
+  ['3', '6', '9'], 
+  ['1', '5', '9'], 
+  ['3', '5', '7']
+];
+  let stepCount = 0;
+  const getStepCount = () => stepCount;
+  const recordGame = () => stepCount++;
+  const checkGame = (player) => {
+      return winCombo.some(combo => combo.every(element => player.getPlayerStep().includes(element)));
```

Comment:
This one-liner appears to be too complex for other programmers to understand. Either change this line's formatting or add comments explaining the logic that this line follows